### ISSUE TITLE
Chunks go out of sync on Chrome 87 (Windows)

### ISSUE THREAD

**Issue by Evoludo** [ddj231/Handel#1]
### Description

When playing multiple chunks at the same time, they go out of sync. Tested with and without specific BPMs.

### Steps to reproduce

1.  Create several chunks.
2. Play chunks simultaneously with one having a BPM that is a multiple of another, with different instruments.

### Expected behaviour

Chunks stay in-sync with each other. Hi-hat is heard at exactly twice the speed of chords.

### Observed behaviour 

Chunks skew out-of-sync over time. Hi-hat is not running at exactly twice the speed of chords.

### Environment

* OS: Windows 10 1803
* Browser: Google Chrome 87.0.4280.88

### Test data

Slightly modified [a-thinker.txt](https://github.com/ddj231/Handel/blob/master/Examples/a-thinker.txt) from Examples at commit [c87e40e](https://github.com/ddj231/Handel/commit/c87e40e920ace9c455ced84ea850b288c01ceb85). Added explicit bpm for non-hihat parts.

```
start
	chunk main
    	play D3, F3, G3, B3 for 1b
       	play F3, A3, C3, E3 for 1b
       	play C3, E3, G3, B3 for 1b
       	play D4, F3, G4, B3 for 1b
       	play F4, G3, B4, D3 for 1b
       	play D4, F3, G4, B3 for 1b
       	play F4, G3, B4, D3 for 1b
       	play D4, F3, G4, B3 for 1b
    endchunk
    
    chunk kickdrums 
    	play E1 for 1b
        rest for 3b
    endchunk
    
    chunk hats 
    	play E4 for 1b
    endchunk
    
    run main with bpm 140, sound piano, loop for 10
    run kickdrums with bpm 140, sound kick, loop for 20
    run hats with bpm 280, sound hihat, loop for 200
    
finish
```

**Comment by ddj231** [comment]
@Evoludo Thanks for bringing up this issue. I was able to recreate it on my end. The Handel interpreter uses Tone.js for scheduling and playing notes. But because there is only one Tone.Transport per page, changing the bpm of the Transport would change it for all tracks.So instead of using Tone.js for setting bpms, the note lengths are calculated based on the bpm and amount of beats, and scheduled in seconds.  I think there may be a floating point rounding issue here but will look into the bug further.

**Comment by ddj231** [comment]
This bug is now addressed in commit 36ce729
Version: 0.1.17

Rather than a rounding issue, the main issue was caused by ignoring the Tone.Transport.bpm in scheduling. The current solution sets the Tone.Transport.bpm to 1000. All run chunks then have their playbackRate set as partBPM/TransportBPM. And note lengths and scheduled times are set accordingly. Setting the playbackRate as the means of setting a part's bpm is the recommended solution by Tone.js. So I believe Handel is now using the best practice.

Notes: 
* chunks with BPMs over the Transport's BPM (1000) will not sync. The Transport BPM can be increased, but there is a tradeoff of slightly less precise scheduling. I think it is best to wait to find the best default value (and later allow users to increase that upper threshold as a configuration passed into the ```RunHandel``` function)
